docs/getting-started/fundamentals-block-development/registration-of-a-block.md

```diff
@@ -61,14 +61,13 @@ Although registering the block also on the server with PHP is still recommended
 
 The function takes two params:
 
--   `$blockNameOrMetadata` (`string`|`Object`) – block type name (supported previously) or the metadata object loaded from the `block.json` file with a bundler (e.g., webpack) or a custom Babel plugin.
+-   `$blockNameOrMetadata` (`string`|`Object`) – block type name (if previously registered on the server) or the metadata object loaded from the `block.json` file with a bundler (e.g., webpack) or a custom Babel plugin.
```

Comment:
The block can be registered only on the client, so I'm not sure whether the rephrased part fits in that case. Otherwise, it's valid.

Reply:
The text in parenthesis `(if previously registered on the server)` meant to clarify that registering a block in the client only using `block type name` only works when the block's metadata has been previously registered on the server.

But I think I can remove this clarification as this is already explained at the beginning of the `Registration of the block with JavaScript (client-side)` section

> When the block is registered on the server, you only need to register the client-side settings on the client using the same block’s name.